charts/hono/Chart.yaml

```diff
@@ -15,7 +15,7 @@ name: hono
 description: |
   Eclipse Hono™ provides remote service interfaces for connecting large numbers of IoT devices to a back end and
   interacting with them in a uniform way regardless of the device communication protocol.
-version: 1.4.2
+version: 1.4.3
```

Comment:
the chart version needs to be bumped once more because there have been other changes in the meantime. Once this is done, we can merge the PR.

Reply:
done!